WolvenKit.App/ViewModels/Documents/RedDocumentViewModel.cs

```diff
@@ -312,8 +314,31 @@ protected void AddTabForRedType(RedBaseClass cls)
         }
     }
 
+    private void OnPartNameChanged(object sender, EventArgs e)
+    {
+        if (GetMainFile() is not RDTDataViewModel m || m.Chunks.Count == 0 || m.Chunks[0] is not { ResolvedData: inkTextureAtlas } cvm ||
+            cvm.Properties.FirstOrDefault((p) => p.Name == "slots") is not ChunkViewModel child)
+        {
+            return;
+        }
+
+        foreach (var chunkViewModel in child.Properties.Where(p => p.ResolvedData is inkTextureSlot).ToList())
+        {
+            m.DirtyChunks.Add(chunkViewModel);
+        }
+
+        SetIsDirty(m.DirtyChunks.Count > 0);
+    }
+
+
     public void PopulateItems()
     {
+        var tab = TabItemViewModels.FirstOrDefault((tab) => tab is RDTInkTextureAtlasViewModel);
```

Comment:
what if there are more than one?
should probably be:
foreach tab in TabViewModels
   if (tab is RDTInkTextureAtlasViewModel inkTextureTab)
        {
            inkTextureTab.ChangeEvent -= OnPartNameChanged;
        }